webpack/assets/javascripts/react_app/components/common/Slot/Slot.test.js

```diff
@@ -0,0 +1,199 @@
+import toJson from 'enzyme-to-json';
+import React from 'react';
+import IntegrationTestHelper from '../../../common/IntegrationTestHelper';
+
+import Fill from '../Fill';
+import Slot from './';
+import FillReducer from '../Fill/FillReducer';
+
+const combinedReducers = { extendable: FillReducer };
+const SlotComponent = ({ text }) => <div>{text}</div>; // eslint-disable-line
+
+jest.unmock('../../../../services/ExtendableRegistry');
+
+describe('Slot-Fill', () => {
+  const integrationTestHelper = new IntegrationTestHelper(combinedReducers);
+
+  it('multiple fills', () => {
+    const wrapper = integrationTestHelper.mount(
+      <React.Fragment>
+        <Fill id="slot-1" fillId="some-key-1" weight={100}>
+          <div key={1}> should be the second in the snap </div>
+        </Fill>
+        <Fill id="slot-1" fillId="some-key-2" weight={200}>
+          <span key={2}> Should be the first in the snap</span>
+        </Fill>
+
+        <Slot id="slot-1" multi>
+          <div> defaul value </div>
+        </Slot>
+      </React.Fragment>
+    );
+
+    wrapper.update();
+    expect(toJson(wrapper.find('Slot'))).toMatchSnapshot();
```

Comment:
Don't we want to take a snapshot of the `state` and the called `actions` to make sure the whole lifecycle worked well?

Reply:
Absolutely ! I added a full redux's lifecycle test